proto/types/audit/audit.pb.go

```diff
@@ -1,6 +1,6 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
-// 	protoc-gen-go v1.27.1
+// 	protoc-gen-go v1.28.1
```

Comment:
can we use protoc-gen-go v1.27.1 to maintain consistency and not upgrade as part of this feature.

Reply:
Still figuring out on how to generate proto files with previous version, not marking this as resolved yet
@akshay196 @niravparikh05 do let me know if you have any context of this